coalib/bears/requirements/PythonRequirement.py

```diff
@@ -83,6 +103,15 @@ def multiple(cls, *args):
                     reqs.append(cls(name, version),)
                 except ValueError:
                     raise TypeError('The tuple must have 2 elements.')
+            elif isinstance(requirement, list):
+                if len(requirement) == 1:
+                    reqs.append(cls(requirement),)
+                else:
+                    try:
+                        name, version = requirement
+                        reqs.append(cls(name, version),)
+                    except ValueError:
+                        raise TypeError('The list must have max 2 elements.')
```

Comment:
You could try turning the `elif` branch for tuples into just an `else` and catch any errors when unpacking the requirement into `name, version`. It would make the whole interface a bit easier for the developer using this. Basically, I want two options for every element in `args`: 1) Requirement without version, 2) Requirement with version. It should also work when I am passing generators, e.g. (('a', 'b') for _ in range(2)), to `multiple`